spec.html

```diff
@@ -42653,6 +42633,21 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-validatetypedarraycompatibility" type="abstract operation">
+        <h1>
+          ValidateTypedArrayCompatibility (
+            _dest_: a TypedArray,
+            _src_: a TypedArray,
```

Comment:
`src` and `dest` don't exactly make sense for this generic AO. `a` and `b` are fine.